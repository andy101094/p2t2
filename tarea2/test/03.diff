44c44
< 44>3 no pertenece a cad.
---
> 44>3 pertenece a cad.
54a55,60
> ==9581== 192 bytes in 32 blocks are definitely lost in loss record 1 of 1
> ==9581==    at 0x4C2E8E9: operator new[](unsigned long) (vg_replace_malloc.c:423)
> ==9581==    by 0x401FD6: info_a_texto(rep_info*) (info.cpp:56)
> ==9581==    by 0x403557: imprimir_cadena(rep_cadena*) (cadena.cpp:422)
> ==9581==    by 0x401A3F: main (principal.cpp:289)
> ==9581== 
