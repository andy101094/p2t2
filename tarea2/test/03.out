1>#  prueba de siguiente, anterior, es_inicio_cadena, es_final_cadena, remover_de_cadena, insertar_antes, pertenece, longitud.
2>Insertado al final.
3>Insertado al final.
4>Insertado al final.
5>Insertado al final.
6>Insertado al final.
7>loc al inicio.
8>loc al siguiente.
9>(2,b)
10>loc al anterior.
11>(1,a)
12>loc al siguiente.
13>loc al siguiente.
14>(3,c)
15>loc al siguiente.
16>loc válido.
17>loc al final.
18>loc al anterior.
19>(4,d)
20>loc al siguiente.
21>(5,e)
22>loc al siguiente.
23>Localizador no válido.
24>loc no válido.
25>(1,a)(2,b)(3,c)(4,d)(5,e)
26>loc al inicio.
27>loc es incio de cad.
28>loc no es final de cad.
29>loc al final.
30>loc no es incio de cad.
31>loc es final de cad.
32>(1,a)(2,b)(3,c)(4,d)(5,e)
33>Insertado al final.
34>loc en la posición 3.
35>Removido.
36>(1,a)(2,b)(4,d)(5,e)(4,e)
37>loc al inicio.
38>Removido.
39>(2,b)(4,d)(5,e)(4,e)
40>loc al final.
41>Removido.
42>(2,b)(4,d)(5,e)
43>Longitud: 3
44>3 no pertenece a cad.
45>loc al inicio.
46>Insertado antes de loc.
47>loc en la posición 3.
48>Insertado antes de loc.
49>(1,a)(2,b)(3,c)(4,d)(5,e)
50>Longitud: 5
51>1 pertenece a cad.
52>10 no pertenece a cad.
53>(1,a)(2,b)(3,c)(4,d)(5,e)
54>Fin.
